Source/Charts/Charts/PieRadarChartViewBase.swift

```diff
@@ -243,7 +243,7 @@ open class PieRadarChartViewBase: ChartViewBase
 
         viewPortHandler.restrainViewPort(offsetLeft: offsetLeft, offsetTop: offsetTop, offsetRight: offsetRight, offsetBottom: offsetBottom)
     }
-
+    
```

Comment:
You're adding in whitespace. Make sure you have the two boxes checked in Xcode Text Editing > White Editing prefs to avoid this.